packages/host/.eslintrc.js

```diff
@@ -73,5 +73,14 @@ module.exports = {
         'qunit/no-conditional-assertions': 'off',
       },
     },
+    {
+      // typescript-eslint recommends turning off no-undef for Typescript files since
+      // Typescript will better analyse that:
+      // https://github.com/typescript-eslint/typescript-eslint/blob/5b0e577f2552e8b2c53a3fb22edc9d219589b937/docs/linting/Troubleshooting.mdx#i-get-errors-from-the-no-undef-rule-about-global-variables-not-being-defined-even-though-there-are-no-typescript-errors
+      files: ['**/*.ts', '**/*.gts'],
+      rules: {
+        'no-undef': 'off',
```

Comment:
I had linter issues that was saying `HTMLElementTagNameMap` (that I introduced in the stack component) does not exist - I was checking how to address this and people online often recommend to turn off `no-undef`.  I also found this exact rule in `boxel-ui` package and copied it over here.